code/modules/clothing/suits/toggles.dm

```diff
@@ -35,12 +38,15 @@
 /obj/item/clothing/suit/hooded/proc/RemoveHood()
 	src.icon_state = "[initial(icon_state)]"
 	suittoggled = FALSE
-	if(ishuman(hood.loc))
-		var/mob/living/carbon/H = hood.loc
-		H.transferItemToLoc(hood, src, TRUE)
-		H.update_inv_wear_suit()
-	else
-		hood.forceMove(src)
+	if(hood)
+		if(ishuman(hood.loc))
+			var/mob/living/carbon/human/H = hood.loc
+			H.transferItemToLoc(hood, src, TRUE)
+			H.update_inv_wear_suit()
+		else
+			hood.forceMove(src)
```

Comment:
don't move it if alternative_mode is on,

also rename alternative_mode to qdel_on_hide_hood or something

Could also be its own PR but whatever